### DISCUSSION TITLE
Check g2.dat version on startup and display warning if outdated.

### DISCUSSION THREAD

**Discussion by ocalhoun6** [OpenRCT2/OpenRCT2#15865] [Feature requests] [answered]
For some reason, the g2.dat file being outdated seems to be a fairly common problem. I've seen it come up several times in bug reports like #15863. And to be fair, it really does look like a bug if you're experiencing it and you don't know why.

Maybe there should be a system in the code that checks the version of the g2.dat file on startup, and if it's not current then display a warning message like "Your g2.dat file seems to be out of date. This could cause font corruption or other problems. Please update g2.dat" ... maybe also with some instructions on how to do so. (And of course, let players dismiss the error message and play the game anyway if they want to.)

**Comment by Gymnasiast** [comment, marked as answer]
Has now been implemented.

**Reply by psuee5** [reply]
I've received this error message trying to install the latest update. How do I fix the issue?

<img width="722" height="508" alt="image" src="https://github.com/user-attachments/assets/707511c4-e889-4519-ad0c-a5bf7902fdc2" />

**Reply by Gymnasiast** [reply]
Quickest way is to uninstall OpenRCT2 and then reinstall. This will keep your parks intact, but I'd recommend backing up the Documents\OpenRCT2 folder beforehand just in case.
